RANSAC_points[] data
coeff[] coeff